perfkitbenchmarker/benchmark_spec.py

```diff
@@ -116,6 +116,62 @@ def RedirectGlobalFlags(self):
     with flag_util.FlagDictSubstitution(FLAGS, lambda: self.config.flags):
       yield
 
+  def ConstructVirtualMachine(self, group_name, group_spec):
```

Comment:
I think this name is a bit misleading (because there is a C"onstructVirtualMachines" I would assume a method named "ConstructVirtualMachine" only operates on one VM at a time). Maybe change it so that it's clear this creates a single *group* of VMs